assets/responsive.css

```diff
@@ -38,8 +39,8 @@ media-querys table
     background: url(./imgs/background.jpg) no-repeat center center;
   }
 
-  #works {
-    width: 63%;
+  .works-container {
+    width: 68%;
```

Comment:
## [OPTIONAL]
- [ ] I think it would be nice if you implement the desktop version without teaching the mobile one. I can see the mobile version is modified because you add the desktop version.I know you can do it let's go :muscle: :clap: